extensions/interactions/GraphInput/directives/GraphInput.js

```diff
@@ -265,6 +265,17 @@ oppia.directive('graphViz', [
             ADD_VERTEX: 2,
             DELETE: 3
           };
+
+          var EDGE_HELP_TEXT = {
+            INITIAL: 'Touch the initial vertex of the edge',
```

Comment:
Can you put periods at the end of all of these?

Reply:
Done